central/complianceoperator/v2/benchmarks/datastore/datastore_impl.go

```diff
@@ -3,14 +3,48 @@ package datastore
 import (
 	"context"
 
-	"github.com/stackrox/rox/central/complianceoperator/v2/benchmarks/store/postgres"
+	benchmarkPostgres "github.com/stackrox/rox/central/complianceoperator/v2/benchmarks/store/postgres"
 	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	postgresSchema "github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/search"
+	pgSearch "github.com/stackrox/rox/pkg/search/postgres"
 )
 
 var _ DataStore = (*datastoreImpl)(nil)
 
 type datastoreImpl struct {
-	store postgres.Store
+	store benchmarkPostgres.Store
+	db    postgres.DB
+}
+
+type ControlResult struct {
+	Control  string `db:"compliance_control"`
+	RuleId   string `db:"compliance_rule_id"`
+	Standard string `db:"compliance_standard"`
+	RuleName string `db:"compliance_rule_name"`
+}
+
+func (d datastoreImpl) GetControlByRuleName(ctx context.Context, ruleNames []string) ([]*ControlResult, error) {
```

Comment:
Here we have two options, either get Controls from the Benchmark Store, or from the Rule store.
I think rule store fits better here.